salt/client/ssh/wrapper/state.py

```diff
@@ -32,6 +32,60 @@
 log = logging.getLogger(__name__)
 
 
+def _ssh_state(chunks, __opts__, __context__, __pillar__, __salt__, st_kwargs,
```

Comment:
I'm very confused by this. Why are we taking in `__opts__` and `__context__` as arguments for this function when they should already be available in this namespace?

Reply:
And `__pillar__`. Can we fix this, @Ch3LL ?